test/fuzz/corpus/README.md

```diff
@@ -0,0 +1,87 @@
+# Corpra for fuzz tests with fixed API calls scenarios
+These corpra contain UR API calls in a predefined order described below.
+All scenarios begin with single calls to urInit() and urAdapterGet().
+
```

Comment:
could you please add some info how to create new/update existing corpras...? perhaps some link to a guide or something

Reply:
https://github.com/google/fuzzing/blob/master/tutorial/libFuzzerTutorial.md#seed-corpus

There's still some work to be done to make a good, minimized, seed corpus. But that's work for later. At that point, we should definitely expand this documentation.